<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/pages/슬라이드5_2.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="backgounrd_1">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>450</y>
      <width>381</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:1000px</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backgounrd_2">
    <property name="geometry">
     <rect>
      <x>569</x>
      <y>460</y>
      <width>381</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:1000px</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backgounrd_3">
    <property name="geometry">
     <rect>
      <x>971</x>
      <y>456</y>
      <width>381</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:1000px</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backgounrd_4">
    <property name="geometry">
     <rect>
      <x>1368</x>
      <y>456</y>
      <width>381</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:1000px</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="print_photo">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>810</y>
      <width>311</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="non_print_photo">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>810</y>
      <width>311</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius : 5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="photo_1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>460</y>
      <width>361</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="photo_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>470</y>
      <width>361</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="photo_3">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>460</y>
      <width>361</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="photo_4">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>460</y>
      <width>361</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
